fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/RemoveAggregationFromAggTable.java

```diff
@@ -51,9 +52,6 @@ public RemoveAggregationFromAggTable() {
     @Override
     public boolean check(OptExpression input, OptimizerContext context) {
         OptExpression scanExpression = input.getInputs().get(0);
-        if (!(scanExpression.getOp() instanceof LogicalOlapScanOperator)) {
```

Comment:
why remove this? how could ensure this operator is LogicalOlapScanOperator?

Reply:
The pattern check has ensure it.